### ISSUE TITLE
Write and read into neo4j using scala languages 

### ISSUE THREAD

**Issue by sarahberraadi** [aribech/SCALA-NEO4J-#1]
Are you using neo4j rest api?

**Comment by aribech** [comment]
No. U have to add some libraries in ur sbt fileSent from my Huawei phone-------- Original message --------From: sarahberraadi <notifications@github.com>Date: Fri, Jul 10, 2020, 21:54To: aribech/SCALA-NEO4J- <SCALA-NEO4J-@noreply.github.com>Cc: Subscribed <subscribed@noreply.github.com>Subject: [aribech/SCALA-NEO4J-] Write and read into neo4j using scala languages  (#1)
Are you using neo4j rest api?
